packages/core/src/mls/EventHandler/EventHandler.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {EventHandlerResult, EventHandlerParams} from './EventHandler.types';
+import {CONVERSATION_EVENT} from '@wireapp/api-client/lib/event';
+import {handleConversationEvent} from './ConversationEvent';
+
+const handleBackendEvent = async (params: EventHandlerParams): EventHandlerResult => {
+  const event = params.event;
+  if (event.type in CONVERSATION_EVENT) {
+    return handleConversationEvent(params);
+  }
+  return Promise.resolve(undefined);
```

Comment:
```suggestion
  return undefined;
```

It's already an async function, it's going to wrap the value in promise.resolve for you